ARTstor.js

```diff
@@ -1,131 +1,484 @@
 {
-	"translatorID": "5278b20c-7c2c-4599-a785-12198ea648bf",
+	"translatorID": "7a690951-ed76-43f5-b761-707fef0345d4",
 	"label": "ARTstor",
-	"creator": "Sebastian Karcher",
-	"target": "^https?://library\\.artstor\\.org",
-	"minVersion": "3.1",
+	"creator": "John Justin, Charles Zeng",
+	"target": "\\.artstor|\\.sscommons\\.org:?\\w*\\/(open)*library",
```

Comment:
I still don't understand why the regex that @aurimasv originally suggested, i.e.
 "^https?://([^/]+\\.)?(artstor|sscommons)\\.org\/(open)?library" doesn't work. That's signficantly more restrictive and more efficient than what you have (which, again, in its current form matches every URL that contains ".artstor" anwhere in the URL).